src/extension.ts

```diff
@@ -142,6 +143,14 @@ export async function activate(extensionContext: ExtensionContext): Promise<any>
         OdoImpl.Instance.loadWorkspaceComponents(event);
     });
 
+    void ComponentTypesView.instance.getAllComponents();
+
+    ComponentTypesView.instance.subject.subscribe((value: string) => {
```

Comment:
This code is not needed.

` void ComponentTypesView.instance.getAllComponents();` call will send `refresh` event for the subject. If there is an instance of RegistryViewer it gets 'refresh' event and going to be updated.